### ISSUE TITLE
problem with ForenSynth_LaDeDa checkpoints

### ISSUE THREAD

**Issue by Giazinga** [barcavia/RealTime-DeepfakeDetection-in-the-RealWorld#3]
Hi authors,
I tried to replicate your experiments on the WildRF dataset.
Using the checkpoints finetuned on WildRF I have any problem to replicate the results in Table 5 of the paper.
But I have found a mismatch in the results of Table 3 when I test using the checkpoint "ForenSynth_LaDeDa.pth". 
Accuracy values in your table 3:
reddit: 74.7
facebook: 70.3
twitter: 59.9

Accuracy in my test (without finetuning)
reddit: 63.5
facebook: 52.2
twitter: 49.6

Could be possible it is the wrong pth file? Since the code seems to work well with the other checkpoints?

**Comment by barcavia** [comment]
HI,
I guess it's because the NPR repo (in which we use for training on the ForenSynth dataset), multiply the NPR representation by 2/3, as you can see here:
![image](https://github.com/user-attachments/assets/cdf42c93-c9f3-4823-87f4-7f06b5fb972d)

That's should solve it :)

**Comment by Giazinga** [comment]
Thank you, with this modification now it seems to work properly
